app/assets/javascripts/darkswarm/cookies_policy/cookies_policy.html.haml

```diff
@@ -0,0 +1,177 @@
+  %h2
+    {{ 'legal.cookies_policy.statistics_cookies' | t}}
+  %p
+    {{ 'legal.cookies_policy.statistics_cookies_desc' | t}}
+
+  - if Spree::Config.cookies_policy_ga_section
+    %p
+      {{ 'legal.cookies_policy.statistics_cookies_analytics_desc' | t}}
+  - if Spree::Config.cookies_policy_matomo_section
+    %p
+      {{ 'legal.cookies_policy.statistics_cookies_matomo_desc' | t}}
+
+  %table{ng: { controller:"CookiesPolicyModalCtrl"}}
+    %tr
+      %th
+        {{ 'legal.cookies_policy.cookie_name' | t}}
+      %th
+        {{ 'legal.cookies_policy.cookie_domain' | t}}
+      %th
+        {{ 'legal.cookies_policy.cookie_desc' | t}}
+    - if Spree::Config.cookies_policy_ga_section
+      %tr
+        %td
+          __utma
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_analytics_utma_desc' | t}}
+      %tr
+        %td
+          __utmt
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_analytics_utmt_desc' | t}}
+      %tr
+        %td
+          __utmb
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_analytics_utmb_desc' | t}}
+      %tr
+        %td
+          __utmc
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_analytics_utmc_desc' | t}}
+      %tr
+        %td
+          __utmz
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_analytics_utmz_desc' | t}}
+    - if Spree::Config.cookies_policy_matomo_section
+      %tr
+        %td
+          _pk_ref, _pk_cvar, _pk_id and _pk_ses
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_matomo_basics_desc' | t}}
+      %tr
+        %td
+          _pk_hsr
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_matomo_heatmap_desc' | t}}
+      %tr
+        %td
+          piwik_ignore
+        %td
+          {{ instance_hostname }}
+        %td
+          {{ 'legal.cookies_policy.cookie_matomo_ignore_desc' | t}}
+
+%h2
+  {{ 'legal.cookies_policy.disabling_cookies_header' | t}}
+%p
+  {{ 'legal.cookies_policy.disabling_cookies_desc' | t}}
+%ul
+  %li
+    %a{ href: "{{'legal.cookies_policy.disabling_cookies_firefox_link' | t}}", target: "_blank" }
+      Firefox
+  %li
+    %a{ href: "{{ 'legal.cookies_policy.disabling_cookies_chrome_link' | t}}", target: "_blank" }
+      Chrome
+  %li
+    %a{ href: "{{ 'legal.cookies_policy.disabling_cookies_ie_link' | t}}", target: "_blank" }
+      Internet Explorer
+  %li
+    %a{ href: "{{ 'legal.cookies_policy.disabling_cookies_safari_link' | t}}", target: "_blank" }
+      Safari
+%p
+  {{ 'legal.cookies_policy.disabling_cookies_note' | t}}
+
+%a.close-reveal-modal{"ng-click" => "$close()"}
+  %i.ofn-i_009-close
```

Comment:
sorry I forgot, if we continue with the engine, all assets should also be in its folder.

Reply:
👍